docs/src/main/tut/streaming.md

```diff
@@ -0,0 +1,432 @@
+---
+layout: default
+title:  "Streaming"
+section: "data"
+source: "https://github.com/non/cats/blob/master/data/src/main/scala/cats/data/Streaming.scala"
+scaladoc: "#cats.data.Streaming"
+---
+
+# Streaming
+
+The `Streaming` data type provides support for sequences of values
+which can be computed on demand. It is immutable (meaning its
+contents will never change), and supports optional memoization.
+
+The data type which `Streaming` implements is often called a *stream*.
+In fact, `Streaming` is similar to an existing Scala data type called
+`Stream` (the name `Streaming` was chosen to avoid a conflict with the
+standard library).
+
+Sometimes the `Streaming` documentation will refer to a *stream*. In
+these cases, we will use lowercase *stream* as a general term,
+distinguished from `Stream` (the specific type from the standard
+library) or `Streaming` (the specific type from Cats).
+
+## Introduction
+
+A non-empty `Streaming` instance is structured like a `List`: it has a
+*cons* cell containing a single value, as well as a reference to a tail
+which will the subsequent values (if any). This means that adding
```

Comment:
There is something missing here. Maybe the word "contain" or the word "hold" or something. I'm not quite sure what the best word is to describe for Streaming, but the sentence as stands is missing something.